_targets.R

```diff
@@ -16,30 +16,126 @@ library(targets)
 library(tarchetypes)
 # Read the tar_script() help file for details.
 
-## Source all functions
-source("R/setup.R")
+tar_option_set(packages = desc::desc_get_deps()$package)
+## Folders
+dir.create("data", showWarnings = FALSE)
 
 ## Load packages
 tar_option_set(packages = .packages())
 
+## To debug a target set the target:
+#      tar_option_set(debug = "lha_events_by_date")
+## and run:
+#      tar_make(callr_function = NULL)
+##  You can do:
+#      tar_make(names = "lha_events_by_date", shortcut = TRUE, callr_function = NULL)
+##  to only run the target of interest and skip checking upstream targets
 
+## Source functions
+r_files <- list.files("R", pattern = "*.R", full.names = TRUE)
+dump <- lapply(r_files, source, echo = FALSE, verbose = FALSE)
+
+future::plan(future::multisession(workers = 6))
+
+## Create output directories:
+hw_output_dir <- "out/heatwave_summaries"
+dir.create(hw_output_dir, showWarnings = FALSE, recursive = TRUE)
 
 # Load --------------------------------------------------------------------
 # time variables
-time_vars <- list(
-  tar_target(start_date, as.Date('1981-01-01')),
-  tar_target(end_date, Sys.Date())
+static_vars <- list(
+  tar_target(start_date, as.Date('1990-04-01')),
```

Comment:
That's right, happy to discuss more. Though this has made me realize that I think I should be generating a full year's worth of surfaces to pass into `generate_pixel_climatologies` (which calls `heatwaveR::ts2clm()`) so that it has the appropriate data at the beginning and end of the seasonal window.

Reply:
Addressed in https://github.com/bcgov/climate-disturbances/pull/10/commits/53b0897e7f40c9a639bb24b57a9526f3c62787c2